catroid/src/androidTest/java/org/catrobat/catroid/test/merge/MergeProjectsTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.test.merge;
+
+import android.content.Context;
+
+import org.catrobat.catroid.MockUtil;
+import org.catrobat.catroid.content.Project;
+import org.catrobat.catroid.formulaeditor.UserList;
+import org.catrobat.catroid.formulaeditor.UserVariable;
+import org.catrobat.catroid.io.XstreamSerializer;
+import org.catrobat.catroid.merge.MergeProjects;
+import org.catrobat.catroid.test.utils.TestUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static junit.framework.Assert.assertFalse;
+
+import static org.junit.Assert.assertTrue;
+
+@RunWith(JUnit4.class)
+public class MergeProjectsTest {
+	private Project firstProject;
+	private Project secondProject;
+	private MergeProjects mergeTask;
+
+	@Before
+	public void setUp() {
+		Context mockContext = MockUtil.mockContextForProject();
```

Comment:
When context is needed please decide which type of test this should be -> if you need androidTest here may use `	Context context = InstrumentationRegistry.getTargetContext();` 

Also think about if a Unit test would be enough -> Move it to test and there is already the MockUtil for UnitTests you copied here